CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
 # Changelog
+## 1.2.0
```

Comment:
we're removing things so this needs to be `2.0.0` rather than `1.2.0`

Reply:
for example, if I were using composer or similar to describe dependencies for a project that used api-raml and I had specified  `~1.1` then I'd be surprised that when the lockfile was updated and the upgrade to `1.2` was made that article vor v3 support was removed. The project depended on it but had been too lazy (or busy ;) to handle any deprecation warnings.

`~1.1` wouldn't upgrade to `2.0` however.